tests/example_experiments_test.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=invalid-name,no-self-use
+import os
+import pyhocon
+
+from deep_qa.common.params import Params, replace_none
+from deep_qa.models import concrete_models
+
+from .common.test_case import DeepQaTestCase
+
+
+class TestExampleExperiments(DeepQaTestCase):
+    def setUp(self):
+        super(TestExampleExperiments, self).setUp()
+        self.example_experiments_dir = "./example_experiments"
+        self.entailment_dir = self.example_experiments_dir + "/entailment"
+        self.memory_networks_dir = self.example_experiments_dir + "/memory_networks"
+        self.multiple_choice_qa_dir = self.example_experiments_dir + "/multiple_choice_qa"
+        self.reading_comprehension_dir = self.example_experiments_dir + "/reading_comprehension"
+        self.sentence_selection_dir = self.example_experiments_dir + "/sentence_selection"
+        self.sequence_tagging_dir = self.example_experiments_dir + "/sequence_tagging"
+
+    def test_entailment_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.entailment_dir)
+
+    def test_memory_network_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.memory_networks_dir)
+
+    def test_multiple_choice_qa_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.multiple_choice_qa_dir)
+
+    def test_reading_comprehension_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.reading_comprehension_dir)
+
+    def test_sentence_selection_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.sentence_selection_dir)
+
+    def test_sequence_tagging_examples_can_be_parsed(self):
+        self.check_experiment_type_can_be_parsed(self.sequence_tagging_dir)
+
+    def check_experiment_type_can_be_parsed(self, directory):
+        for param_file in os.listdir(directory):
+            param_path = os.path.join(directory, param_file)
+            param_dict = pyhocon.ConfigFactory.parse_file(param_path)
+            params = Params(replace_none(param_dict))
+            model_class = concrete_models[params.pop("model_class")]
+            # Tests will try to create root directories as we have /net/efs paths,
+            # so we just remove the serialisation aspect here.
+            params["model_serialization_prefix"] = None
+            self.get_model(model_class, params)
```

Comment:
Oh, actually this won't work.  You need to have some things set from the data file before the model can be built.  You'd need to mangle all of the train and validation files, too, and make sure that the correct files were written to `self.TRAIN_FILE`, etc...

This might be more work than it's worth.  It _would_ be really nice to have tests that our example files pass, though.  A better option might be to have some fixtures checked in to the repo for the tests, and have the examples point to those toy fixture files.